diff --git a/configure.in b/configure.in
--- configure.in
+++ configure.in
@@ -5090,15 +5090,29 @@
 if test -n "$with_system_mythes" && test "$with_system_mythes" != "no"; then
     AC_MSG_RESULT([external])
     SYSTEM_MYTHES=YES
-    AC_CHECK_HEADER(mythes.hxx, [],
-       [ AC_MSG_ERROR(mythes.hxx headers not found.)], [])
-    AC_CHECK_LIB(mythes, main, [],
-        [ AC_MSG_ERROR(mythes library not found.)], [])
+    AC_LANG_PUSH([C++])
+    PKG_CHECK_MODULES(MYTHES, mythes, MYTHES_PKGCONFIG=yes, MYTHES_PKGCONFIG=no)
+    if test "$MYTHES_PKGCONFIG" = "no"; then
+        AC_CHECK_HEADER(mythes.hxx, [],
+            [ AC_MSG_ERROR(mythes.hxx headers not found.)], [])
+        AC_CHECK_LIB(mythes-1.2, main, [],
+            [ MYTHES_FOUND=no], [])
+ 	if test "$MYTHES_FOUND" = "no"; then
+            AC_CHECK_LIB(mythes, main, [MYTHES_FOUND=yes],
+                [ MYTHES_FOUND=no], [])
+	fi
+	if test "$MYTHES_FOUND" = "no"; then
+		AC_MSG_ERROR([mythes library not found!.])
+	fi
+    fi
+    AC_LANG_POP([C++])
 else
     AC_MSG_RESULT([internal])
     SYSTEM_MYTHES=NO
 fi
 AC_SUBST(SYSTEM_MYTHES)
+AC_SUBST(MYTHES_CFLAGS)
+AC_SUBST(MYTHES_LIBS)
 
 dnl ===================================================================
 dnl Checking for lpsolve
diff --git a/set_soenv.in b/set_soenv.in
--- set_soenv.in
+++ set_soenv.in
@@ -1878,6 +1878,8 @@
 ToFile( "SYSTEM_HYPH",       "@SYSTEM_HYPH@",      "e" );
 ToFile( "HYPHEN_LIB",        "@HYPHEN_LIB@",       "e" );
 ToFile( "SYSTEM_MYTHES",     "@SYSTEM_MYTHES@",    "e" );
+ToFile( "MYTHES_CFLAGS",     "@MYTHES_CFLAGS@",    "e" );
+ToFile( "MYTHES_LIBS",       "@MYTHES_LIBS@",      "e" );
 ToFile( "PAM",               "@PAM@",              "e" );
 ToFile( "NEW_SHADOW_API",    "@NEW_SHADOW_API@",   "e" );
 ToFile( "PAM_LINK",          "@PAM_LINK@",         "e" );
diff --git a/solenv/inc/libs.mk b/solenv/inc/libs.mk
--- solenv/inc/libs.mk
+++ solenv/inc/libs.mk
@@ -342,7 +342,11 @@
 .ELSE
 HUNSPELLLIB=-lhunspell-1.2
 .ENDIF
+.IF "$(SYSTEM_MYTHES)" == "YES"
+MYTHESLIB=$(MYTHES_LIBS)
+.ELSE
 MYTHESLIB=-lmythes
+.ENDIF
 PYUNOLIB=-lpyuno
 LPSOLVELIB=-llpsolve55
 SOFFICELIB=-lsofficeapp
