--- src/Makefile.am	2010-06-09 18:28:59.000000000 +0000
+++ src/Makefile.am.working	2011-06-09 00:39:46.000000000 +0000
@@ -13,13 +13,6 @@
 spec_dir = $(top_srcdir)/spec
 spec_files := $(patsubst $(spec_dir)%.xml,_generated%.py,$(wildcard $(spec_dir)/*.xml))
 
-BUILT_SOURCES = \
-	_generated/interfaces.py \
-	_generated/constants.py \
-	_generated/errors.py \
-	_generated/__init__.py \
-	$(spec_files)
-
 generateddir = $(telepathydir)/_generated
 nodist_generated_PYTHON = $(BUILT_SOURCES)
 
@@ -28,8 +21,7 @@
 nodist_version_PYTHON = _version.py
 
 CLEANFILES = \
-	_version.py \
-	$(BUILT_SOURCES)
+	_version.py 
 
 EXTRA_DIST = _version.py.in
 
