diff -urN NetworkManager-0.6.3-orig/src/backends/NetworkManagerArch.c NetworkManager-0.6.3/src/backends/NetworkManagerArch.c
--- NetworkManager-0.6.3-orig/src/backends/NetworkManagerArch.c	2006-03-27 20:30:57.000000000 +0200
+++ NetworkManager-0.6.3/src/backends/NetworkManagerArch.c	2006-06-08 17:49:52.000000000 +0200
@@ -952,3 +952,14 @@
 {
 }
 
+
+/*
+ * nm_system_get_mtu
+ *
+ * Return a user-provided or system-mandated MTU for this device or zero if
+ * no such MTU is provided.
+ */
+guint32 nm_system_get_mtu (NMDevice *dev)
+{
+	return 0;
+}
