--- a/configure.ac	2008-07-18 16:30:17.000000000 +0200
+++ b/configure.ac	2012-01-24 13:14:26.000000000 +0100
@@ -72,29 +72,7 @@
 	esac
 	AC_TRY_CFLAGS([$TRY_CFLAGS $CFLAGS],[OPT_CFLAGS="$TRY_CFLAGS"]);;
     ppc-* | powerpc-*)
-	have_altivec=no
-	for TRY_CFLAGS in "-mpim-altivec -force_cpusubtype_ALL" -faltivec -maltivec -fvec; do
-	    AC_TRY_CFLAGS([$OPT_CFLAGS $TRY_CFLAGS $CFLAGS],
-		[save_CFLAGS="$CFLAGS"
-		 CFLAGS="$OPT_CFLAGS $TRY_CFLAGS $CFLAGS"
-		 AC_MSG_CHECKING([if <altivec.h> is needed])
-		 AC_TRY_COMPILE([],
-		    [typedef vector int t;
-		     vec_ld(0, (unsigned char *)0);],
-		    [have_altivec=yes; AC_MSG_RESULT(no)],
-		    [AC_TRY_COMPILE([#include <altivec.h>],
-			[typedef vector int t; vec_ld(0, (unsigned char *)0);],
-			[AC_DEFINE([HAVE_ALTIVEC_H],,
-			    [Define to 1 if you have the <altivec.h> header.])
-			 have_altivec=yes; AC_MSG_RESULT(yes)],
-			[AC_MSG_RESULT(unsupported)])])
-		 CFLAGS="$save_CFLAGS"])
-	    if test "$have_altivec" = "yes"; then
-		ARCH_OPT_CFLAGS="$TRY_CFLAGS"
-		AC_DEFINE([ARCH_PPC],,[ppc architecture])
-		break
-	    fi
-	done;;
+	have_altivec=no;;
     sparc-* | sparc64-*)
 	AC_DEFINE([ARCH_SPARC],,[sparc architecture])
 	TRY_CFLAGS="$OPT_CFLAGS -mcpu=ultrasparc -mvis"
